src/v/redpanda/admin/api-doc/migration.def.json

```diff
@@ -10,7 +10,35 @@
         },
         "ns": {
             "type": "string",
-            "description": "Topic namespace, if not present it is assumed that topic is in kafka namespace"
+            "description": "Topic namespace. If not present it is assumed that topic is in the \"kafka\" namespace"
+        }
+    }
+},
+"mountable_topic": {
+    "type": "object",
+    "properties": {
+        "topic_location": {
+            "type": "string",
+            "description": "Unique topic location in cloud storage with the format <topic name>/<cluster_uuid>/<initial_revision>"
```

Comment:
@nvartolomei could I double check that `initial_revision` is a placeholder that is meant to be replaced by some value? What is the "initial revision" and where can one find this value? Or is the path parameter just the string `initial_revision`?

Reply:
It is indeed a placeholder. It is a number assigned to a topic on creation, it can be helpful to disambiguate between topics created with the same name. For a live topic (i.e. the one available in the cluster) AFAIK the only admin API endpoint that shows it is `/v1//cloud_storage/anomalies/{namespace}/{topic}/{partition}`. For a detached (unmounted) topic it can be found using the functionality added in this PR. It can also be visible in logs.